### ISSUE TITLE
typing_extensions.OrderedDict is not recognized as generic

### ISSUE THREAD

**Issue by JelleZijlstra** [microsoft/pyright#2869]
```python
from typing_extensions import OrderedDict

d: OrderedDict[int, int]
```
produces `Expected no type arguments for class "OrderedDict"` with current Pylance.

`OrderedDict` is the only instance of `_Alias()` in typing_extensions, so maybe there's some missing handling for it.

**Comment by erictraut** [comment]
Pyright's logic assumes that the file that defines the symbol imports from the file that contains the alias. In this case, `typing_extensions.pyi` doesn't contain an import from `collections`. If I add the statement `import collections`, it works.

I realize that Pyright might be making a tenuous assumption here. If you don't like the idea of adding the import statement in typeshed, I can look at ways to eliminate this assumption.

**Comment by erictraut** [comment]
I've eliminated this assumption, so it now works.

**Comment by JelleZijlstra** [comment]
Thanks! I'd be OK with changing typeshed too; I think we have a collections.abc import in typing.pyi that mypy needs for similar reasons.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.210, which I just published. It will also be included in the next release of pylance.
